README.md

```diff
@@ -15,6 +16,18 @@ depend on the number of vertices, the number of edges, or both?
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
 
+1. Adjacency list to matrix 
+- this conversion is dependent on the number of edges. The number of edges in the graph will reflect the size of our adjacency list, since each nested list in the adjacency list corresponds to an edge in the graph, therefore this conversion is dependent on the number of edges, not the number of vertices
+- the outer loop goes through each of the entry in the adjacency list to get a time complexity $O(E)$
+- the inner nested loop goes through each edge (nested list) of the given vertice, and pushes a 1 to that position in the converted matrix. All other entries will be 0. Since we evaluate an edge for every nested list, the time complexity for the inner loop is dependent on the number of edges therefore its complexity is also $O(E)$
+- the total time complexity would then be $O(E^2)$ where $E$ is the number of edges in the graph
```

Comment:
That doesn't seem quite right -- remember what we talked about in class today? The algorithm can't iterate over all edges for each vertex.